exporter/awsxrayexporter/internal/translator/cause.go

```diff
@@ -486,6 +488,56 @@ func fillDotnetStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 	return exceptions
 }
 
+func fillGoStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	var line string
+	var label string
+	var path string
+	var lineNumber int
+
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	_, _ = r.ReadLine()
+	exception := &exceptions[0]
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		if strings.Contains(line, "goroutine") {
```

Comment:
Is this safe? What if a package or method name includes `goroutine`?  This should probably use a regex to ensure that the correct action is taken.